desc/objectives/_geometry.py

```diff
@@ -567,6 +576,12 @@ def __init__(
         self._surface_grid = surface_grid
         self._plasma_grid = plasma_grid
         self._use_softmin = use_softmin
+        self._use_signed_distance = use_signed_distance
+        if use_softmin and use_signed_distance:
```

Comment:
can you explain a bit more why these two don't work together? I'm not sure I totally understand

Reply:
use argmin on distances to determine sign after using softmin